### PR TITLE
Show language selection screen on first launch (AIC-76, AIC-465)

### PR DESCRIPTION
**Pull request by sam33rdhakal**
Changes:
- Displays language settings in between the video animation
- Refactors info module (moved language settings classes into localization_ui module)
- Fixes welcome screen's greeting message issue [AIC-465]

### CODE REVIEW

File: splash/src/main/kotlin/edu/artic/splash/SplashActivity.kt [hunk 1]
```diff
@@ -1,41 +1,54 @@
...
 
     override val viewModelClass: KClass<SplashViewModel>
         get() = SplashViewModel::class
 
+    override val refreshLanguageUponChange = false
+
     private lateinit var fadeInAnimation: ViewPropertyAnimator
     private var errorDialog: AlertDialog? = null
 
+    private var height: Int = 0
```
[the remark this answers is not in the document]
Comment (sam33rdhakal) [reply splash/src/main/kotlin/edu/artic/splash/SplashActivity.kt]: Removed.

File: ui/src/main/kotlin/edu/artic/ui/BaseActivity.kt [hunk 2]
```diff
@@ -49,11 +52,13 @@ abstract class BaseActivity : AppCompatActivity(), HasSupportFragmentInjector {
     override fun onCreate(savedInstanceState: Bundle?) {
         if (useInjection) {
             AndroidInjection.inject(this)
-            languageSelector.currentLanguage
-                    .subscribe {
-                        ensureConfigIncludesAppLocale()
-                        recreate()
-                    }.disposedBy(disposeBag)
+            if (refreshLanguageUponChange) {
```
[the remark this answers is not in the document]
Comment (sam33rdhakal) [reply ui/src/main/kotlin/edu/artic/ui/BaseActivity.kt]: Renamed `refreshLanguageUponChange` to `shouldRecreateUponLanguageChange`.

File: localization_ui/src/main/java/artic/edu/localization/ui/LanguageSettingsViewModel.kt [hunk 3]
```diff
@@ -18,17 +19,18 @@ import javax.inject.Inject
...
     init {
+        appLocale.onNext(languageSelector.getAppLocale())
 
-        selectedLanguage.onNext(languageSelector.getAppLocale())
-
-        selectedLanguage
+        appLocale
                 .distinctUntilChanged()
                 .subscribe {
                     analyticsTracker.reportEvent(
```
[the remark this answers is not in the document]
Comment (sam33rdhakal) [reply localization_ui/src/main/java/artic/edu/localization/ui/LanguageSettingsViewModel.kt]: Updated.

File: localization_ui/src/main/AndroidManifest.xml [hunk 4]
```diff
@@ -0,0 +1,2 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="artic.edu.localization.ui" />
```
Comment (Cliabhach) [inline localization_ui/src/main/AndroidManifest.xml]: Rename module to `edu.artic.....`

Comment (sam33rdhakal) [reply localization_ui/src/main/AndroidManifest.xml]: Updated.

**Pull request merged**